client/a8c-for-agencies/style.scss

```diff
@@ -16,6 +16,12 @@
 	--color-sidebar-text-alternative: #bbe0fa;
 	--color-sidebar-gridicon-fill: #bbe0fa;
 
+	--color-scary-0: var(--studio-red-0);
```

Comment:
To properly style the destructive menu items with the right color, we must define these colors.